doc/source/glossary.rst

```diff
@@ -7,38 +7,42 @@ Glossary
 
    Resource
      An entity representing anything in your infrastructure that you will
-     associate metric(s) with. It is identified by a unique ID and can
+     associate |metric|\ (s) with. It is identified by a unique ID and can
```

Comment:
should there be a random space between \ and (s)

Reply:
that is to make | separated with (, but there should not be space there. Putting a \ before it makes it a functional space.